pow/src/main/java/tech/pegasys/teku/pow/Eth1DepositManager.java

```diff
@@ -50,16 +50,13 @@ public Eth1DepositManager(
     this.minimumGenesisTimeBlockFinder = minimumGenesisTimeBlockFinder;
   }
 
+  @SuppressWarnings("FutureReturnValueIgnored")
```

Comment:
not a huge fan of this any ideas @ajsutton

Reply:
Will look through the PR properly in the morning, but yeah that's definitely wrong.  Need to find which future is being ignored and shove a `.finish` call on it at least to report any exceptions.